Automattic/wp-calypso
# Emails: Remove old Email Comparison page

This pull request retires the former `Email Comparison` page, which we no longer use:

Old | New
------ | -----
![01](https://user-images.githubusercontent.com/594356/116910481-fb135080-ac45-11eb-8635-99b7cfe79a73.png) | ![02](https://user-images.githubusercontent.com/594356/116911765-922cd800-ac47-11eb-9a33-791c0f987bfc.png)

It also addresses [this comment](https://github.com/Automattic/wp-calypso/pull/46280#discussion_r593160507) by updating the list of features to use the right function to handle widows.

#### Testing instructions

1. Run `git checkout remove/old-email-comparison-page` and start your server, or open a [live branch](https://calypso.live/?branch=remove/old-email-comparison-page)
2. Log into a WordPress.com account that has a domain without email
3. Open the [`Domains` page](http://calypso.localhost:3000/domains/manage)
4. Click the `Add` button for that domain
5. Assert that the new `Email Comparison` page looks and works as before
6. Change the width of your browser, and assert that no widows appear in the list of features
